scalyr_agent/scalyr_client.py

```diff
@@ -220,6 +233,9 @@ def __init__(
         # flag to disable new addEvents format, TODO: remove this when we are confident it works
         self.__disable_logfile_addevents_format = disable_logfile_addevents_format
 
+        # whether or not to monotonically increase event timestamps
```

Comment:
"whether or not to enforce monotonically increasing event timestamps within the same session."